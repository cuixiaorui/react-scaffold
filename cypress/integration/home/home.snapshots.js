describe('home page', () => {
  it('snapshots', () => {
    cy.visit('/').then(() => {
      cy.get('[data-testid=home]').toMatchSnapshot();
    });
  });
});
