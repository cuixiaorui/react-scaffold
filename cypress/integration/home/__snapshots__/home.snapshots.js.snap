exports[`home page > snapshots #0`] = `
<div data-testid="home">
  <div data-testid="home-nav" class="sc-bdVaJa fqDwql">
    <a data-testid="home-login-button" href="#/body">入门</a>
  </div>
  <div>
    <h1 data-testid="home-title">react scaffold</h1>
    <p data-testid="home-describe">这是一个前端 react 脚手架</p>
  </div>
</div>
`;
